cmd/test-utils_test.go

```diff
@@ -366,6 +367,20 @@ func StartTestStorageRPCServer(t TestErrHandler, instanceType string, diskN int)
 	testRPCServer.AccessKey = credentials.AccessKey
 	testRPCServer.SecretKey = credentials.SecretKey
 
+	// Create dummy format json file.
+	emptyConfig := []byte("{}")
```

Comment:
The real meaning wrt to current logic is, `{}` works which is bad though.  This should be fixed.  Once this is fixed the test case should be fixed else it fails to work.

I can add a `FIXME` me to tell why empty json is used.